### ISSUE TITLE
Submenus for the sub pages

### ISSUE THREAD

**Issue by gunchleoc** [widelands/widelands_website#8]
Submenus for the various areas of the page must be added. We had them on the old page, but they were subtle to notice and generally, I'd like another layout.

**Comment by gunchleoc** [comment]
(by sirver)
My attempt to this can currently be found on alpha.widelands.org. I changed the site layout to a two column layout (which might change again as more boxes are added). I moved the navigation to the top and added subnavigation (only an example for the Wiki Page). I tried to keep the navigation as unobtrusive as possible; the submenus only appear after the user hovers over the main link. I need feedback on this.
1) Is the layout okay? 
2) should the main link be included as additional sublink (that's the way it is now: wiki links to wiki_index and the first submenu entry also)
3) How does this look with different browsers. 

I'm unsure if this is a good solution to the navigation problem....
